packages/main/src/DayPicker.js

```diff
@@ -580,39 +689,77 @@ class DayPicker extends UI5Element {
 	}
 
 	_handleItemNavigationBorderReach(event) {
-		const currentMonth = this._month,
-			currentYear = this._year;
-		let newMonth,
-			newYear,
-			newDate,
-			currentDate;
-
-		if (event.end) {
-			currentDate = new Date(this._weeks[this._weeks.length - 1][event.offset].timestamp * 1000);
-			newMonth = currentMonth < 11 ? currentMonth + 1 : 0;
-			newYear = currentMonth < 11 ? currentYear : currentYear + 1;
-			newDate = currentDate.getMonth() === newMonth ? currentDate.getDate() : currentDate.getDate() + 7;
-		} else if (event.start) {
-			currentDate = new Date(this._weeks[0][event.offset].timestamp * 1000);
-			newMonth = currentMonth > 0 ? currentMonth - 1 : 11;
-			newYear = currentMonth > 0 ? currentYear : currentYear - 1;
-			newDate = currentDate.getMonth() === newMonth ? currentDate.getDate() : currentDate.getDate() - 7;
+		const currentItem = this._itemNav._getCurrentItem();
+		let newDate,
```

Comment:
Please choose only one way of defining variables. Either 
```js
let a,
    b,
    c;
```
or
```js
let a;
let b;
let c;
```

and follow that convention.

(Refer to line 520)

Reply:
I've chosen the more explicit way of definition.